packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextAnchorViewManager.java

```diff
@@ -49,8 +49,8 @@ public void setAccessible(ReactTextView view, boolean accessible) {
   }
 
   // maxLines can only be set in master view (block), doesn't really make sense to set in a span
-  @ReactProp(name = ViewProps.NUMBER_OF_LINES, defaultInt = ViewDefaults.NUMBER_OF_LINES)
-  public void setNumberOfLines(ReactTextView view, int numberOfLines) {
+  @ReactProp(name = ViewProps.MAXIMUM_NUMBER_OF_LINES, defaultInt = ViewDefaults.MAXIMUM_NUMBER_OF_LINES)
+  public void setMaxNumberOfLines(ReactTextView view, int numberOfLines) {
```

Comment:
Hmm, same question as earlier. Are we changing behavior of the existing prop? Seems like this removes the old one in favor of the new one (breaking change).

Reply:
Not really as on the js side it's just mapped. 
On Fabric we already use that new name and I need to make it consistent across arch and platforms.  https://github.com/facebook/react-native/blob/247a913af000aa5912ef9acfca8a834a17c28f59/packages/react-native/ReactCommon/react/renderer/components/text/ParagraphProps.cpp#L81